projects/fsastorefrontlib/src/cms-components/my-account/address-info/address-form/address-form.component.html

```diff
@@ -0,0 +1,158 @@
+<form
+  (ngSubmit)="verifyAddress()"
+  [formGroup]="addressForm"
+  class="info-container p-4"
+>
+  <div class="form-group" formGroupName="country">
```

Comment:
Each form-group should have row class too, even if it contains single element inside. This is not only for consistency with other My Documents forms, but also if in the future we add more elements inside that row wrapper.

Reply:
Yes, I saw that, but there is difference in html structure between Address info and Personal details component. To maintain consistency I need to refactor html structure first..